### PR TITLE
Test engine fix

### PR DESCRIPTION
**Pull request by taojintianxia**
this PR should fix some spelling issue and remanagement the document.

### CODE REVIEW

File: document/current/content/features/test-engine/_index.cn.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ chapter = true
 
 ShardingSphere提供了完善的测试引擎。它以XML方式定义SQL，每条SQL由SQL解析单元测试引擎和整合测试引擎驱动，每个引擎分别为H2、MySQL、PostgreSQL、SQLServer和Oracle数据库运行测试用例。
 
+为了使测试更容易上手，shardingsphere 中的测试引擎不必修改任何 **Java** 代码，只需要修改相应的配置文件即可运行断言
```
Comment (terrymanu) [inline document/current/content/features/test-engine/_index.cn.md]: Is there need a `.` to end of the sentence?

File: document/current/content/features/test-engine/sql-case.cn.md [hunk 2]
```diff
@@ -5,15 +5,20 @@ title = "SQL测试用例"
 weight = 1
 +++
 
-## 配置
+## 目标
+
+SQL测试用例的代码位于 `sharding-sql-test` 模块下。该测试用例的作用主要有两个：
+  
+  1. 通过单元测试，测试通配符的替换以及 `SQLCasesLoader` 的稳定性
+  2. 将SQL测试用例中 `resource` 下定义的所有 SQL 共享给其他项目
```
Comment (terrymanu) [inline document/current/content/features/test-engine/sql-case.cn.md]: `resource` or `resources`?

File: document/current/content/features/test-engine/_index.cn.md (hunk 1 above)
Comment (taojintianxia) [reply document/current/content/features/test-engine/_index.cn.md]: so there shoud not be a full stop at the end of a sentence if the setence is the last one?

Comment (taojintianxia) [reply document/current/content/features/test-engine/_index.cn.md]: followings should be a guideline

**Pull request merged**